docs/deployments/environment-variables.mdx

```diff
@@ -17,18 +17,45 @@ Enter the name and value of your variable. All values are encrypted, and can onl
 
 It is also possible to specify what environments the variable should be available in.
 
+<Warning>
+After creating or updating an environment variable, you will need to redeploy your project for the changes to take effect.
+</Warning>
+
 ## Accessing environment variables
 
-Within your Dart code, you can access environment variables using the `Platform.environment` map.
+Within your Dart code, you can access environment variables using the `Platform.environment` map. Any package that uses this map can access Globe environment variables.
 
 ```dart
 String? value = Platform.environment['MY_VARIABLE'];
 ```
 
+### Using "dotenv" package
+
+If you prefer to use a `.env` file for local development, you can use the [dotenv package](https://pub.dev/packages/dotenv) to load environment variables from a file locally and use Globe environment variables when deployed.
+
+Add the `includePlatformEnvironment: true` to the `DotEnv` constructor to include platform environment variables when `.env` file is not defined.
+
+```dart
+final env = DotEnv(includePlatformEnvironment: true)..load();
+String? value = env['MY_VARIABLE'];
+```
+
+### Using "envied" package
+
+Currently the Globe environment variables does not support the [envied package](https://pub.dev/packages/envied). This is because the Globe environment variables are not available during the build process, and `envied_generator` requires it. We are currently working on adding support for it.
+
+Check [this issue](https://github.com/invertase/globe/issues/59) for more information. Upvote if this is something you would like to see.
```

Comment:
```suggestion
```

Reply:
I'll be fixing this very soon, so not worth adding this not just to remove it.

Reply:
Removed